Libraries/LibWeb/HTML/HTMLBaseElement.idl

```diff
@@ -7,6 +7,6 @@ interface HTMLBaseElement : HTMLElement {
     [HTMLConstructor] constructor();
 
     [CEReactions] attribute USVString href;
-    [CEReactions, Reflect] attribute DOMString target;
+    [CEReactions, Reflect, ReflectURL] attribute DOMString target;
```

Comment:
Thanks, fixed it

Reply:
There are still a few places where we have `[Reflect, ReflectURL]` at the moment. I'm not sure this is necessary because `[ReflectURL]` should be able to be used in place of `[Reflect]`.

Reply:
Also, should `HTMLBaseElement.target` have the `ReflectURL` attribute on it? According to the spec it is expected to be a [valid target name or keyword](<https://html.spec.whatwg.org/multipage/document-sequences.html#valid-navigable-target-name-or-keyword>).